admin/controllers/restart.js

```diff
@@ -145,15 +144,6 @@ controller.postSubmitRestartList = async (req, res, next) => {
   const pupilUrlSlugs = pupilsRestarted && pupilsRestarted.map(p => encodeURIComponent(p.urlSlug))
   const pupilsToHighlight = pupilUrlSlugs.join()
   req.flash('info', restartInfo)
-
```

Comment:
These status recalculations have moved to the service layer and will be deleted once the new pupil state work is complete.